app/src/main/java/com/example/myweatherapp/mvp/presenter/MyWeatherAppPresenter.kt

```diff
@@ -14,7 +14,9 @@ class MyWeatherAppPresenter(
         model.getData(CITY)
             .subscribeOn(Schedulers.io())
             .observeOn(AndroidSchedulers.mainThread())
-            .subscribe({ data -> view.showData(data) }, { view.showError() })
+            .subscribe(
```

Comment:
Does this have the correct indentation?

Reply:
![image](https://user-images.githubusercontent.com/73836200/107376184-5b23ac00-6ac8-11eb-9549-5683a64bbfbe.png)

at least that is what happens when i do a ctrl L

Reply:
Okey its good for me